examples/lifelong_learning/atcii/ll_atcii_tutorial.md

```diff
@@ -0,0 +1,440 @@
+            name:  train-worker
+            imagePullPolicy: IfNotPresent
+            args: ["train.py"]  # training script
+            env:  # Hyperparameters required for training
+              - name: "early_stopping_rounds"
+                value: "100"
+              - name: "metric_name"
+                value: "mlogloss"
+    trigger:
+      checkPeriodSeconds: 60
+      timer:
+        start: 02:00
+        end: 24:00
+      condition:
+        operator: ">"
+        threshold: 500
+        metric: num_of_samples
+  evalSpec:
+    template:
+      spec:
+        nodeName: $EVAL_NODE
+        dnsPolicy: ClusterFirstWithHostNet
+        containers:
+          - image: $CLOUD_IMAGE
+            name:  eval-worker
+            imagePullPolicy: IfNotPresent
+            args: ["eval.py"]
+            env:
+              - name: "metrics"
+                value: "precision_score"
+              - name: "metric_param"
+                value: "{'average': 'micro'}"
+              - name: "model_threshold"  # Threshold for filtering deploy models
+                value: "0.5"
+  deploySpec:
+    template:
+      spec:
+        nodeName: $INFER_NODE
+        dnsPolicy: ClusterFirstWithHostNet
+        containers:
+        - image: $CLOUD_IMAGE
+          name:  infer-worker
+          imagePullPolicy: IfNotPresent
+          args: ["inference.py"]
+          env:
+          - name: "UT_SAVED_URL"  # unseen tasks save path
+            value: "/ut_saved_url"
+          - name: "infer_dataset_url"  # simulation of the inference samples 
+            value: "/data/testData.csv"
+          volumeMounts:
+          - name: utdir
+            mountPath: /ut_saved_url
+          - name: inferdata
+            mountPath: /data/
+          resources:  # user defined resources
+            limits:
+              memory: 2Gi
+        volumes:   # user defined volumes
+          - name: utdir
+            hostPath:
+              path: /lifelong/unseen_task/
+              type: DirectoryOrCreate
+          - name: inferdata
+            hostPath:
+              path:  /data/
+              type: DirectoryOrCreate
+  outputDir: "/output"
+EOF
+```
+## 1.5 Check Lifelong Learning Job
+**(1). Query lifelong learning service status**
+
+```
+kubectl get lifelonglearningjob atcii-classifier-demo
+```
+
+**(2). View pods related to lifelong learning job**
+
+```
+kubectl get pod
+```
+
+**(3). Process unseen tasks samples**
+
+In a real word, we need to label the hard examples in our unseen tasks which storage in `UT_SAVED_URL` with annotation tools and then put the examples to `Dataset`'s url. By this way, we can realize the function of updating models based on the data generated at the edge.
+
+**(4). View result files**
+
+Artifacts including multi-task learning models, partitioned sample sets, etc. can be found in `outputDir`, and the inference result is stored in the `Dataset`'s url.
+
+# 2 Develop and Integrate Customized Modules
```

Comment:
This tutorial not only shows how to run the default example (with or without customized configurations), but also shows how to develop and integrate user-defined modules.